### DISCUSSION TITLE
Is there a way to update the suggestion list?

### DISCUSSION THREAD

**Discussion by HubKing** [google/mozc#572] [Q&A] [answered]
Below is a comparison of suggestions between Mozc (fctix5-mozc in Arch) and Microsoft IME (Windows 11). As you see, Windows contains a lot of names of current famous people than Mozc. I know I can manually add words, but I am not talking about that. What I think is updating Mozc's database so that it would show a lot more and accurate suggestions for the names of current famous people, events, companies, etc. Is there a way? I mean, Mozc is made by Google and Google must already have such names in its database somewhere, judging by Google Search's suggestions.

![image](https://user-images.githubusercontent.com/10900489/164989436-c9509492-3fc0-47cc-baab-38c5da767277.png)
![image](https://user-images.githubusercontent.com/10900489/164989458-b440217d-929a-4f81-8675-531c34e2599e.png)

Windows' default IME.
![image](https://user-images.githubusercontent.com/10900489/164989653-8cdaef32-f41e-4a1f-8892-606d61c7d0e5.png)
![image](https://user-images.githubusercontent.com/10900489/164989616-795a18de-0312-4395-aa36-9b2229bcaa99.png)

Google Search
![image](https://user-images.githubusercontent.com/10900489/164990102-24c8a1a6-75ad-40c1-b9ad-9d4583e17085.png)
![image](https://user-images.githubusercontent.com/10900489/164990067-f486e2d7-b182-45c9-93b9-4b8b45ae0da3.png)

**Comment by hiroyuki-komatsu** [comment, marked as answer]
Hi thank you for the suggestion.

The word dictionary for Mozc is basically created from publicly available data under some open source licenses.
https://github.com/google/mozc/blob/master/src/data/dictionary_oss/README.txt

If there are some additional data for Mozc, we might consider to use those data.

If you want to add some specific words as a part of system dictionary, we accept pull requests to
https://github.com/google/mozc/blob/master/src/data/oss/aux_dictionary.tsv

Thanks,

**Reply by HubKing** [reply]
Well, if Mozc cannot include the same word data as Google Japanese Input due to licensing, I wish it could make it a little bit easier for users to add words and share them. I can see the import menu in user dictionary, but it is too cumbersome to navigate to that UI and add words, let alone sharing them.  Instead, it could be something like users can create a web URL to a text file that contains custom word list, and by "subscribing" to that URL, other users can automatically get any new words added to the online file. This is how [advertisement block lists](https://easylist.to/) work, as far as I know.
